mentat/terminal/themes.py

```diff
@@ -1,4 +1,4 @@
-Themes = {
+Themes: dict[str | None, dict[str, str]] = {
```

Comment:
I would prefer if this was lowercase; I keep expecting Themes to be a class

Reply:
changed